src/components/SignIn.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import signInLogo from '../assets/signInLogo.svg';
+
+const SignIn = () => {
+  return (
+    <div class="bg-transparent">
```

Comment:
wrap the whole thing by a wrapper using the container class

https://tailwindcss.com/docs/container